### DISCUSSION TITLE
Sales Credit Note API

### DISCUSSION THREAD

**Discussion by nncodestone** [fattureincloud/api#391] [Q&A] [answered]
Hi,

Can you please help me to provide me  to get Sales Credit Notes via API? Please let me know the Type of the document to be used in API call. I mean for Invoices we use type = invoice, but for Credit memo , may I know the type value?

Thanks & Regards
Nagesh

**Comment by emajo** [comment, marked as answer]
Hi @nncodestone, I think the type you need is `credit_note`.

Thanks,
Emanuele.

**Reply by valmoz** [reply]
Hi @nncodestone did we answer your question?

Thanks
Mauro
